ocs_ci/ocs/ui/views.py

```diff
@@ -112,7 +116,7 @@
     "ReadOnlyMany": ('input[data-test="Read Only (ROX)-radio-input"]', By.CSS_SELECTOR),
     "pvc_size": ('input[data-test="pvc-size"]', By.CSS_SELECTOR),
     "pvc_create": ('button[data-test="create-pvc"]', By.CSS_SELECTOR),
-    "pvc_actions": ('button[data-test-id="actions-menu-button"]', By.CSS_SELECTOR),
+
```

Comment:
If "button[aria-label='Actions']" works for 4.6 and 4.7, I think it would be better to just substitute it here. If not, you could add it to 4.8 dictionary without removing it in the 4.6/4.7 one. We might want to run regression tests on later versions of 4.7 so it's better not to break the existing tests.

Reply:
added